spec/features/assets/asset_submission_spec.rb

```diff
@@ -7,12 +7,124 @@
   let(:study) { create(:study) }
   let(:request_factory) { :sequencing_request }
   let(:asset) { create(:library_tube) }
+  let(:flowcell_types) { create_list(:flowcell_type, 2) }
   let(:request_types) { create_list(:sequencing_request_type, 2) }
   let(:original_request_type) { request_types.first }
   let(:selected_request_type) { original_request_type }
   let(:selected_read_length) { '76' }
+  let(:request_flowcell_type_validators) do
+    RequestType::Validator.create(
+      request_type: selected_request_type,
+      request_option: 'requested_flowcell_type',
+      valid_options: flowcell_types.map(&:name)
+    )
+  end
   let!(:original_request) do
-    create(request_factory, study: study, project: project, asset: asset, request_type: original_request_type)
+    create(request_factory, study: study, project: project, asset: asset, request_type: selected_request_type)
+  end
+
+  describe 'The request form does not set default values' do
+    let(:user) { create(:admin) }
+
+    before do
+      login_user user
+      visit labware_path(asset)
+      click_link 'Request additional sequencing'
+    end
+
+    describe 'when the form is loaded' do
+      it 'does not set request type' do
+        expect(page).to have_select('Request type', selected: 'Select a request type')
+      end
+    end
+
+    describe 'when the user selects a request type' do
+      before { select 'Request Type 1', from: 'Request type' }
+
+      it 'does not set flowcell type to default value' do
+        expect(page).to have_select('Flowcell type', selected: 'Select a requested flowcell type')
+      end
+
+      it 'does not set read length to default value' do
+        expect(page).to have_select('Read length', selected: 'Select a read length')
+      end
+    end
+  end
+
+  describe 'Validation of Flowcell Type field for illumina-HTP NovaSeq requests' do
+    let(:user) { create(:admin) }
+
+    before do
+      allow(selected_request_type).to receive(:validator_for).with('requested_flowcell_type').and_return(
+        request_flowcell_type_validators
+      )
+      login_user user
+      visit labware_path(asset)
+      click_link 'Request additional sequencing'
+    end
+
+    it 'displays an error if request type is of type illumina_htp_novaseq and Flowcell Type is not set' do
```

Comment:
Hi @sabrine33 

Am I right in saying this doesn't actually use a 'illumina-HTP NovaSeq request', it uses a test request type made by the factory - but you're mocking the 'validator' behaviour against that request type so it should behave in the same way?

I think it's OK but could you change the wording so it doesn't say that you're using illumina-HTP NovaSeq requests, and add a comment to explain what you're doing?

Thanks 🙏